exec/java-exec/src/main/java/org/apache/drill/exec/expr/stat/ParquetFilterPredicate.java

```diff
@@ -18,5 +18,17 @@
 package org.apache.drill.exec.expr.stat;
 
 public interface ParquetFilterPredicate<T extends Comparable<T>> {
-  boolean canDrop(RangeExprEvaluator<T> evaluator);
+
+  /**
+   * Define the validity of a row group against a filter
+   * <ul>
+   *   <li>ALL : all rows match the filter (can not drop the row group and prune filter)
+   *   <li>NONE : no row matches the filter (can drop the row group)
+   *   <li>SOME : some rows only match the filter (can not drop the row group not the filter)
+   *   <li>INAPPLICABLE : filter can not be applied
+   * </ul>
+   */
+  enum ROWS_MATCH {ALL, NONE, SOME, INAPPLICABLE}
```

Comment:
It is more common for Drill to use all capital formatting only for enum values, the enum itself uses Camel notation.

Reply:
Done